client/devdocs/docs-example/readme.jsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * External Dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import request from 'superagent';
+import Remarkable from 'remarkable';
+import RemarkableReactRenderer from 'remarkable-react';
+
+/**
+ * Internal Dependencies
+ */
+import Spinner from 'components/spinner';
+import Card from 'components/card';
+import Ribbon from 'components/ribbon';
+
+class Readme extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.md = new Remarkable( {
+			linkify: true, // auto-convert things that look like links to <a> components
+			typographer: true // prettify quotes in the content
+		} );
+		this.md.renderer = new RemarkableReactRenderer();
+	}
+
+	componentWillMount() {
+		this.requestReadme( this.props.component );
+	}
+
+	componentWillReceiveProps( nextProps ) {
+		if ( this.props.component !== nextProps.component ) {
+			this.requestReadme( this.props.component );
+		}
+	}
+
+	requestReadme = ( component ) => {
+		this.setState( {
+			loading: true
+		} );
+		request.get( '/devdocs/service/content' )
```

Comment:
Fetching straight from the display component is discouraged, as is directly making the request.